@types/commercetools-uikit__select-input/index.d.ts

```diff
@@ -45,11 +53,24 @@ declare module '@commercetools-uikit/select-input' {
     | 'tabSelectsValue'
   >;
   const SelectInput: {
-    (props: SelectInputProps): JSX.Element;
+    <OptionType extends OptionTypeBase = { label: string; value: string }>(
+      props: SelectInputProps<OptionType>
+    ): JSX.Element;
     displayName: string;
     defaultProps: Pick<SelectInputProps, 'maxMenuHeight' | 'menuPortalZIndex'>;
     // Static fields
     isTouched: (touched: unknown) => boolean;
+    // Aliases from react-select built-in components
```

Comment:
It's a bit hard to type this: https://github.com/commercetools/ui-kit/blob/75f476984f5fe371765f94cfaa009da507e91205/src/components/inputs/select-input/select-input.js#L271-L275

For now I just type only the components that we use.

Maybe something to consider in general when we decide to expose such things.

/cc @jonnybel